.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: scala
+dist: trusty
+sudo: required
```

Comment:
Is sudo required? It might be nice to try both ways, because while `sudo: required` workers have more RAM, the containerized `sudo: false` ones have quicker startup times, so for smaller jobs it might be beneficial.

Reply:
We don't need to do this now, though… low priority.

Reply:
copy/pasted from `lagom/lagom` and made some changes.

I don't think `sudo` is required.